app/Http/Controllers/PostController.php

```diff
@@ -44,6 +74,23 @@ public function store()
         return view('posts.create');
```

Comment:
It is considered a bad practice in most cases to have a return statement in an if or loop. The reason might be the fact that you are not using the Laravel framework to handle form input validation. I advise you to start doing this. It will not only prevent weird behavior here, but will allow you to validate much more than only required fields and enable users to have their forms prefilled with old data when validation fails.